pages/02_auth_and_sso.md

```diff
@@ -8,27 +8,66 @@ In most embedding scenarios, you will want to enable single sign-on so that the
 
 Additional considerations:
 
 * A common desire is to use a single 'service' account to authenticate the users. This is not a recommended approach, because it does not allow you to apply [data security]({{ site.baseurl }}/pages/04_multitenancy_and_rls) or to track usage on a per-user basis.
-* The trusted ticket is redeemable only once and the Tableau Server session is only valid for the visualization that was originally loaded. Therefore, your web application must request an additional ticket if refreshes the web page or navigates to a different page that contains embedded content.
-* By default, tickets can be redeemed only for visualizations, and not for other content pages in Tableau Server. To enable the user to see those, you must configure [unrestricted tickets](http://kb.tableau.com/articles/issue/login-prompt-when-embedding-server). See also: the [embedding non-view content]({{ site.baseurl }}/pages/06_embedding_non_view_content) page in this playbook.
-* If your web application has dynamic ip addresses, such that it is not feasible to trust a specific set of static ip addresses, you should create a small 'ticket requester' application that only allows requests from your web application, requests tickets from Server, and then returns them to your web application. You can then deploy this 'ticket requester' application to a static ip address.
+* The trusted ticket is redeemable only once and is valid for the current Tableau Server session. The session allows the user to access any of the views that the user would have if they logged onto the server. For more information, see [Trusted Authentication](https://help.tableau.com/current/server/en-us/trusted_auth.htm).
+* By default, tickets can be redeemed only for visualizations, and not for other content pages in Tableau Server. To enable the user to see those, you must configure [unrestricted tickets](https://kb.tableau.com/articles/issue/login-prompt-when-embedding-server). See also: the [embedding non-view content]({{ site.baseurl }}/pages/06_embedding_non_view_content) page in this playbook.
+* If your web application has dynamic IP addresses, such that it is not feasible to trust a specific set of static IP addresses, you should create a small 'ticket requester' application that only allows requests from your web application, requests tickets from Server, and then returns them to your web application. You can then deploy this 'ticket requester' application to a static IP address.
```

Comment:
Change this:
>* If your web application has dynamic IP addresses, such that it is not feasible to trust a specific set of static IP addresses, you should create a small 'ticket requester' application that only allows requests from your web application, requests tickets from Server, and then returns them to your web application. You can then deploy this 'ticket requester' application to a static IP address.

to this:

* If your web application has dynamic IP addresses, such that it is not feasible to trust a specific set of static IP addresses, you have a few options. Either, create a small 'ticket requester' application that only allows requests from your web application, requests tickets from Server, and then returns them to your web application. This 'ticket requester' application can be deployed to a static IP address. Or, consider leveraging one of the other authentication mechanisms listed above that do not depend on IP whitelisting.

Reply:
done